src/main/java/com/synopsys/integration/detect/lifecycle/run/operation/OperationRunner.java

```diff
@@ -363,17 +364,6 @@ public final BdioUploadResult uploadBdioIntelligentPersistent(BlackDuckRunData b
         );
     }
 
-    public final CodeLocationWaitData calculateCodeLocationWaitData(List<WaitableCodeLocationData> codeLocationCreationDatas) throws OperationException {
-        return auditLog.namedInternal("Calculate Code Location Wait Data", () -> new CodeLocationWaitCalculator().calculateWaitData(codeLocationCreationDatas));
-    }
-
-    public final void publishCodeLocationNames(Set<String> codeLocationNames) throws OperationException {
-        auditLog.namedInternal(
-            "Publish CodeLocationsCompleted Event",
-            () -> codeLocationEventPublisher.publishCodeLocationsCompleted(codeLocationNames)
-        );
```

Comment:
Looks like we're walking away from the code locations report in the diagnostic zip. That seems fine since it didn't contain complete results anyway, and that was never an issue. (It may make sense at some point to completely remove the generation of that report from the diagnostic system.)